source/manual/how-to-be-a-good-buddy.html.md

```diff
@@ -0,0 +1,38 @@
+---
+owner_slack: "#govuk-developers"
+title: How to be a good buddy
+section: Learning GOV.UK
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+---
+
+When a new joiner starts on govuk, it's good practice to assign them a "buddy" to help them get started on govuk and it's infrastructure and provide initial support.
+
+## What is a buddy?
+
+A buddy acts as a temporary guide for new joiners, helping them understand the govuk programme and generally get started. Some responsibilities of a buddy can include:
+
+- explaining how our stack works
+- pointing the new joiner to useful docs/slack channels/general resources
+- being on hand to answer specific questions
+- generally make them feel welcome
+
+Ideally, a buddy will be in the same team and sit within the same function as the new joiner eg: a new frontend developer would have a frontend buddy.
```

Comment:
Interesting, I was under the impression we looked for on-programme but off-team for frontenders. Guess I was wrong! With many teams only having 1 FE it's hard to tell!

Reply:
That's a good point. I was under the impression that what I wrote was the ideal but that's also my best guess. I'm open to changing this, do we have prior art/thoughts on this?

Reply:
I'm told backend aim for someone in team, so that's probably worth sticking with.